modules/FP_events/language/en_us.lang.php

```diff
@@ -134,5 +134,6 @@
 	'LBL_NAME_SECOND' => 'Last Name',
      
 	'LBL_ACTIVITY_STATUS' => 'Activity Status',
+	'LBL_FP_EVENT_LOCATIONS_FP_EVENTS_1_FROM_FP_EVENTS_TITLE' => 'Locations',
```

Comment:
Sorry to not be clear on this.
There is a comment in the end of the file for the string `LBL_ERROR_MSG_4`
see:
`//LBL_ERROR_MSG_4 Begins with a number (controller.php line 581) for example 10 emails have failed to send.`
This is intended for translators but its only useful for the ones actually reading the language file.
For those on Crowdin the comments are not visible unless they are included in the same line then the string. 

On this situation the line 105 should be (with the commented line after the comma:

`    'LBL_ERROR_MSG_4' => ' emails have failed to send. Please check that all the contacts you are inviting have a valid email address. (See suitecrm.log)', //LBL_ERROR_MSG_4 Begins with a number (controller.php line 581) for example 10 emails have failed to send. 
`